packages/lesswrong/server/resolvers/tagResolvers.ts

```diff
@@ -447,3 +447,23 @@ export async function updateDenormalizedContributorsList(tag: DbTag): Promise<Co
   
   return contributionStats;
 }
+
+addGraphQLResolvers({
+  Query: {
+    async UserTopTags(root: void, {userId}: {userId: string}, context: ResolverContext) {
+      const tagsRepo = new TagsRepo();
+      const topTags = await tagsRepo.getUserTopTags(userId);
+      return topTags;
+    },
+  },
+});
+
+addGraphQLSchema(`
+  type UserTopTag {
+    tagId: String!
+    name: String!
+    count: Int!
+  }
+`);
+
+addGraphQLQuery('UserTopTags(userId: String!): [UserTopTag!]');
```

Comment:
This should use `defineQuery` - see `migrationsDashboardGraphql.ts` for an example